app/controllers/api/versions_controller.rb

```diff
@@ -8,9 +8,12 @@ class Api::VersionsController < Api::BulkProjectController
   def index
     @max_results = [Integer(params[:max_results] || MAX_RESULTS), MAX_RESULTS].min
 
-    @versions = Version.includes(:project).where(
-      "versions.updated_at > ?",
-      Time.parse(params.require(:since))
-    ).order(:updated_at)
+    @versions = Version
+      .includes(:project)
```

Comment:
Should this be a join? I have a feeling the includes turns the query into a join anyways...

Reply:
Yeah, once you start adding conditions to the includes it'll use a join instead 👍🏻